openquake/commonlib/oqvalidation.py

```diff
@@ -75,9 +75,8 @@ class OqParam(valid.ParamSet):
     intensity_measure_types_and_levels = valid.Param(
         valid.intensity_measure_types_and_levels, None)
     # hazard_imtls = valid.Param(valid.intensity_measure_types_and_levels, {})
-    hazard_investigation_time = valid.Param(valid.positivefloat, None)
     interest_rate = valid.Param(valid.positivefloat)
-    investigation_time = valid.Param(valid.positivefloat, None)
+    investigation_time = valid.Param(valid.positivefloat, 50.)
```

Comment:
The issue is with scenario calculations, where the user should not set an investigation time. Then without a default an error would be raised and the user would be forced to add an otherwise unused value every time.